src/main/java/pages/ForgotPasswordPage.java

```diff
@@ -0,0 +1,27 @@
+    package pages;
+
+    import org.openqa.selenium.By;
+    import org.openqa.selenium.WebDriver;
+
+    public class ForgotPasswordPage {
+        WebDriver driver;
+        private By forgotPasswordField = By.cssSelector("input[class='form-control'] ");
+        private By retrievePasswordButton = By.xpath("//form[@id='form_forgotpassword']//button[@type='submit']");
+        public ForgotPasswordPage(WebDriver driver) {
+            this.driver = driver;
+        }
+        public void enterEmailField(String email){
+            driver.findElement(forgotPasswordField).sendKeys(email);
+        }
+        public void retrievePasswordButton(){
+            driver.findElement(retrievePasswordButton).click();
+        }
+        public Boolean errorShown(){
+            Boolean isDisplaied =  driver.findElement(By.xpath("//div[@class='alert alert-danger']")).isDisplayed();
+            return isDisplaied;
+        }
+
```

Comment:
We have issues with the empty lines across the whole package. Therefore, this comment is to fix these in all documents. Naida left multiple comments for these things, but they still occur. Please fix these issues in all places.